pkg/docker/docker.go

```diff
@@ -114,9 +114,28 @@ type Launchable struct {
 	SuppliedEnvVars map[string]string
 }
 
-func (l *Launchable) Disable() error {
+func (l *Launchable) Disable(gracePeriod time.Duration) error {
```

Comment:
Why can't the `Disable()` functionality be the same between hoist and docker with only `disable()` differing?

Reply:
Because they are different to be terminated, as shown in the commit description, what's the suggestions?

Reply:
I was wondering if the `Disable()` function in the hoist launchable object could more closely reflect the `Disable()` function in the docker launchable object? Specifically asking around the channel usage in the docker disable function, but not in the hoist disable function.